paper/paper.md

```diff
@@ -67,9 +67,9 @@ Mesa is a Python-based framework for ABM that provides a comprehensive set of to
 2. Data collection and support for model experimentation
 3. Visualization systems
 
-
 This decoupled design allows selective use of components while enabling extension and customization.
 
+Mesa follows a two-track development model where new features are first released as experimental before being stabilized. Experimental features are clearly marked as such and may have their APIs change between releases. They are graduated to stable status once their APIs and implementations are proven through community testing and feedback.
```

Comment:
perhaps mention here that mesa uses semantic versioning (this is partly why we have experimental stuff)

Reply:
```suggestion
Mesa follows a two-track development model where new features are first released as experimental before being stabilized. Experimental features are clearly marked as such and may have their APIs change between releases. They are graduated to stable status once their APIs and implementations are proven through community testing and feedback. Stable features follow semantic versioning.
```